src/encoded/static/components/search.js

```diff
@@ -711,30 +711,48 @@ TypeTerm.propTypes = {
     total: PropTypes.number.isRequired,
 };
 
-
 class Facet extends React.Component {
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
 
         // Set initial React commponent state.
         this.state = {
-            facetOpen: false,
+            filteredTerms: null,
         };
 
         // Bind `this` to non-React methods.
-        this.handleClick = this.handleClick.bind(this);
+        this.handleSearch = this.handleSearch.bind(this);
     }
 
-    handleClick() {
-        this.setState(prevState => ({ facetOpen: !prevState.facetOpen }));
+    handleSearch(event) {
+        // search term entered by the user
+        const filterVal = String(event.target.value.toLowerCase().replace(/ /g, '').replace(/[^\w\s]/gi, ''));
```

Comment:
Not a regex expert, but I feel like the \s in the second replace would also take care of deleting spaces done by the first replace. And technically, I doubt you need the `i` in `gi` since you’ve already lowercased the string, though I don't know if that has any real performance difference.

Reply:
I was thinking about this a little more. My original thought had been that if you type "%%%", the list of terms shouldn't be filtered at all so that was why I was ignoring non-alphabetical characters. However, then you don't get an error message if you type "seq%%%" and I feel like you probably should. What do you think makes the most sense here? I am thinking that maybe we should match all non-white space characters including symbols.